lib/winston/transports/file.js

```diff
@@ -558,10 +558,13 @@ File.prototype._createStream = function () {
   })(this._getFile());
 };
 
+File.prototype._getFilename = function () {
+  return this._basename;
+};
```

Comment:
Sure, I could make it a getter - I was looking for consistency in how the other accessors were implemented. I may be missing something though - how is an additive function a breaking change?